docs/docs/docker/tagging-docker-images.mdx

```diff
@@ -38,7 +38,7 @@ Options for `registries` in `pants.toml`:
   registry.
 
 - `repository` - Format the repository part of the image name for this image. See [Setting a
-  repository name](doc:tagging-docker-images#setting-a-repository-name) for details of this option.
+  repository name](./tagging-docker-images.mdx#setting-a-repository-name) for details of this option.
```

Comment:
Totally minor, but I think a fragment only link works, to avoid needing to repeat the current page name?

```suggestion
  repository name](#setting-a-repository-name) for details of this option.
```

Reply:
That's what I would have thought, but I was just copying from the other links on the page. If it should work, I can update the rest 🤷🏽

Reply:
If you want to test, finding the same page in https://github.com/pantsbuild/pantsbuild.org, running `npm start` and editing the page I find is good. https://github.com/pantsbuild/pantsbuild.org/blob/main/docs/docs/docker/tagging-docker-images.mdx

In this case, I already tested, and yes, the fragment does work. But... also the rest of the page is the same, and it works... so no need to change. 🤷

Reply:
> npm start and editing the page I find is good.

This was going to be my next question 😆  I'd never built the docs locally, so wasn't sure if it was this simple - I found some other broken links that I may as well fix after I get back